lib/Wikia/tests/Persistence/User/AttributePersistenceSwaggerTest.php

```diff
@@ -32,12 +32,10 @@ public function setUp() {
 		$this->apiProvider = $this->getMockBuilder( ApiProvider::class )
 			->setMethods( [ 'getAuthenticatedApi' ] )
 			->disableOriginalConstructor()
-			->disableAutoload()
```

Comment:
These `disableAutoload()` calls were causing problems with the unit tests. Classes which were mocked here couldn't be found by later tests. It seems to somehow relate to the DI library we started using. Turns out we don't actually need these calls though, so we're just removing them which fixes it.